crates/op-rbuilder/src/tests/framework/apis.rs

```diff
@@ -2,6 +2,7 @@ use super::DEFAULT_JWT_TOKEN;
 use alloy_eips::{eip7685::Requests, BlockNumberOrTag};
 use alloy_primitives::B256;
 use alloy_rpc_types_engine::{ForkchoiceUpdated, PayloadStatus};
+use http::Uri;
```

Comment:
Have they break compatibility with uri::Uri? 
Can you do .to_string for uri::Uri?